### DISCUSSION TITLE
Authorize.net is deprecated, should it be removed?

### DISCUSSION THREAD

**Discussion by fballiano** [OpenMage/magento-lts#3756] [Questions & Answers]
in the payment method section of the backend, authorize.net methods are marked as deprecated

<img width="390" alt="Screenshot 2024-01-25 alle 16 23 47" src="https://github.com/OpenMage/magento-lts/assets/909743/d7051987-77a7-484b-b8c8-0df3daaad16a">

and if you click this link:
<img width="677" alt="Screenshot 2024-01-25 alle 16 25 13" src="https://github.com/OpenMage/magento-lts/assets/909743/aea39c7e-2d12-4da5-a708-bedde655fce6">

it triggers an error:
<img width="570" alt="Screenshot 2024-01-25 alle 16 25 41" src="https://github.com/OpenMage/magento-lts/assets/909743/8f2ca4ef-436c-41ad-9554-6d02eb1b402e">

From what I see they were deprecated in 1.9.4.2 but info is scarse.

is somebody using this payment methods? do they work? should we remove them from the core?

**Comment by addison74** [comment]
Good observation. Can you turn them into modules for whoever wants to add them after we remove them from the main code?

**Reply by fballiano** [reply]
they're coupled into the core, it's not easy, sincerely at the moment I don't want to do that and, if we remove them, they should be removed only from next (which I hope will be released sometimes or we just wasted a huge amount of time)

**Reply by addison74** [reply]
Let's wait some feedback. I agree for the next branch.

**Comment by sreichel** [comment]
Maybe check M2 code for what has changed?

**Reply by fballiano** [reply]
It seems there's no authorize.net module in M2: https://github.com/search?q=repo%3Amagento%2Fmagento2%20authorize.net&type=code

**Reply by sreichel** [reply]
Just found some post howto set ip up in M2 ... e.g. https://amasty.com/knowledge-base/how-to-set-up-authorize-net-in-magento-2.html

From reading Authorize.net still has an working API. (did not digged deeper and dont know how much work it takes)

**Reply by fballiano** [reply]
I've seen that blog post too but there seem to be nothing in the code, maybe the blog post is old, no idea.
